<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InvoiceReview</class>
 <widget class="QDialog" name="InvoiceReview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Review Τιμολόγησης</string>
  </property>
  <widget class="QTableView" name="tableFintrade">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>256</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableLines">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>511</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateFrom">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2000</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="date">
    <date>
     <year>2000</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
   <property name="displayFormat">
    <string>d/M/yy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateTo">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>d/M/yy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushDelete">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>85</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushEmail">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>50</y>
     <width>85</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Από Ημερομηνία</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Εως Ημερομηνία</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
